pkgs/applications/networking/remote/xrdp/default.nix

```diff
@@ -89,12 +97,18 @@ let
     dontDisableStatic = true;
     configureFlags = [
       "--with-systemdsystemunitdir=/var/empty"
+      "--enable-devel-logging"
```

Comment:
This doesn't seem like something meant for development? Why are we enabling it?

Reply:
I enabled for testing. It actually gives more detail about where that log came from.

https://github.com/neutrinolabs/xrdp/blob/9bbb2ec68f390504c32f2062847aa3d821a0089a/common/log.h#L76

Reply:
It allows you to customize logs at source file level, here is a small patch in my test instance running as a NixOS container
![image](https://github.com/NixOS/nixpkgs/assets/15693688/3ea4e718-6572-4959-a5e3-dcfc4bdff27b)

Reply:
It still feels like something we shouldn't enable by default though.

Reply:
I removed it then